grub2-efi/install

```diff
@@ -11,17 +11,23 @@ case "$target" in
   i?86 ) target=i386 ;;
   x86_64 | amd64 ) target=x86_64 ;;
   aarch64 ) target=arm64 ;;
-  armv7l ) target=arm ;;
+  arm* ) target=arm ;;
 esac
 
 target="$target-efi"
 
 echo "target = $target"
 
+# Systems without efivars should just fall back to install grub as removable
```

Comment:
thanks for explanation, can you please update this pr and summarize your comment into code comment so in future we know why it is needed? thanks